### PR TITLE
Update 0.4.3

### LINKED ISSUE #19: config.simple add denyRule exceptions
Add the option to configure denyRule exceptions in the simple setup.

### PR DESCRIPTION
**Pull request by jbe99**

### CODE REVIEW

File: charts/microgateway/templates/configmap.yaml [hunk 1]
```diff
@@ -91,6 +91,9 @@ data:
         - enable: {{ $simple.mapping.denyRules.enabled }}
           log_only: {{ $simple.mapping.denyRules.logOnly }}
           level: {{ $simple.mapping.denyRules.level }}
+          {{- with $simple.mapping.denyRules.exceptions }}
```
Comment (dieti80) [inline charts/microgateway/templates/configmap.yaml:94]: @jubeergon 
Shouldn't this be "{{ toYaml . | nindent 12 }}"

At least it is configured this way in deployment.yaml. I suggest to use only one way to achieve such things.

File: charts/microgateway/README.md [hunk 2]
```diff
@@ -95,6 +95,7 @@ The following table lists configuration parameters of the Airlock Microgateway c
 | config.simple.backend.port | int | `8080` | Backend port |
 | config.simple.backend.protocol | string | `"http"` | Backend protocol |
 | config.simple.mapping.denyRules.enabled | bool | `true` | Enable all Deny rules. |
+| config.simple.mapping.denyRules.exceptions | list | `[]` | Specification of deny rule exceptions. |
```
Comment (dieti80) [inline charts/microgateway/README.md]: Change "Specification of deny rule exceptions." to "Deny rule exceptions" or "Deny rule exceptions configuration."

In my opinion it is not a specification, but a configuration or setting, ...

**Review by dieti80 — CHANGES_REQUESTED** [review]

File: charts/microgateway/templates/configmap.yaml (hunk 1 above)
Comment (jbe99) [reply charts/microgateway/templates/configmap.yaml:94]: Then we would have to change the apps: as well. 
https://github.com/ergon/airlock-helm-charts/blob/9b2e1de4782156ba7cf27563ace35b14b9e02349/charts/microgateway/templates/configmap.yaml#L77

I copied it from there. 
But yes agree consistent would be better.

File: charts/microgateway/README.md.gotmpl [hunk 3]
```diff
@@ -192,13 +192,12 @@ config:
       denyRules:
         exceptions:
           - parameter_name:
-              pattern: ^my_param_name$
-              ignore_case: false
+              pattern: ^content$
+              ignore_case: true
```
Comment (jbe99) [inline charts/microgateway/README.md.gotmpl:196]: ignore_case: true is the Default

**Review by dieti80 — APPROVED** [review]

**Review by dieti80 — APPROVED** [review]

**Pull request merged**